kubernetes/samples/scripts/create-weblogic-domain/domain-home-on-pv/README

```diff
@@ -0,0 +1,37 @@
+This sample has been modified to create a domain on pv and also to update the domain using the new WDT model.
+The updateDomain is only to be used with domain created using WDT
+
+Use case - WDT can be used to discover an domain (could be on-prem). Running wdt discover
```

Comment:
Alternatively can I a domain generated by Operator  in Domain on PV model

Reply:
If the domain is already created by wdt, there must be a model file already. Why use wdt to generate one?